src/OpenReader.m

```diff
@@ -1085,4 +1083,14 @@ - (void)createFolders:(NSMutableArray *)params
     }
 }
 
+
+/**
+ Get the current timestamp
+ 
+ @return current timestamp as a string
+ */
++ (NSString *)currentTimestamp {
```

Comment:
As before, this is a class method and my understanding is properties should only be used for instance variables - at least thats what I think [Apple's document is telling me](https://developer.apple.com/library/content/releasenotes/ObjectiveC/ModernizationObjC/AdoptingModernObjective-C/AdoptingModernObjective-C.html#//apple_ref/doc/uid/TP40014150-CH1-SW4) 😬